src/app/add-item/add-item.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'app-add-item',
+  templateUrl: './add-item.component.html',
+  styleUrls: ['./add-item.component.scss'],
+})
+
+export class AddItemComponent implements OnInit {
+  items = [];
+  // itemList -> chenge next whit the list of lists for choice one
+  itemList = 'example';
+  itemID: number = this.items.length + 1;
+  itemTitle: string;
+  itemDescription: string;
+
+  ngOnInit() {
+    this.itemTitle = '';
+    this.itemDescription = '';
+  }
+
+  addItem() {
+    this.items.push({
+      list: this.itemList,
+      id: this.itemID,
+      title: this.itemTitle,
+      description: this.itemDescription,
+      completed: false,
+    });
+
+    this.itemID++;
+
+    this.ngOnInit();
```

Comment:
No. If you want to clear some field do it with a method having meaningful name like `clearItemData()`